api/src/features/multiflash/multiflash_use_case.py

```diff
@@ -7,15 +7,25 @@
 
 
 class MultiflashResponse(BaseModel):
-    ph_index: List[str] = Field(..., description="Phase labels (vapor, liquid, aqueous, mercury)")
-    ph_frac: List[float] = Field(..., description="Fraction of each phase (sums to 1)")
+    phase_label: List[str] = Field(..., description="Phase labels (vapor, liquid, aqueous, mercury)")
+    phase_fraction: List[float] = Field(..., description="Fraction of each phase (sums to 1)")
     moles: List[List[float]] = Field(..., description="Mole fractions of each phase")
 
     @classmethod
     def from_values(
-        cls, ph_index: npt.NDArray[bytes], ph_frac: npt.NDArray[float], moles: npt.NDArray[float]
+        cls,
+        phase_label: npt.NDArray[npt.NDArray[bytes]],
+        phase_fraction: npt.NDArray[float],
+        moles: npt.NDArray[float],
     ) -> "MultiflashResponse":
-        return cls(ph_index=ph_index.tolist(), ph_frac=ph_frac.tolist(), moles=moles.tolist())
+        new_phase_label = []
+        for entry in phase_label:
+            bytestring = entry.tobytes().decode("utf-8").strip()
+            if bytestring != 7 * "\x00":
+                new_phase_label.append(bytestring)
+            else:
+                new_phase_label.append("")
```

Comment:
Why are we adding empty strings to the list of phases in the result?

Reply:
I guess it's not. Was just to keep the array dimensions consistent. But we only really need the non-empty ones, so I will fix it.